browser/ui/color/brave_color_mixer.cc

```diff
@@ -693,7 +693,7 @@ void AddBraveOmniboxColorMixer(ui::ColorProvider* provider,
                                const ui::ColorProviderKey& key) {
   ui::ColorMixer& mixer = provider->AddMixer();
 
-  mixer[kColorBraveOmniboxResultViewSeparator] = {nala::kColorDividerSubtle};
```

Comment:
Its not safe to use Nala theme variables without checking for a custom theme, unfortunately :cry: